src/core/server/integration_tests/http/versioned_router.test.ts

```diff
@@ -260,20 +260,23 @@ describe('Routing versioned requests', () => {
   });
 
   it.each([
-    ['public', '2022-02-02', '2022-02-03'],
+    ['public', '2023-10-31', ''],
```

Comment:
NIT: Yeah.. this one is awkward.

Reply:
haha yeah, I'll refactor to remove this awkwardness and better capture what is going on there.